gslib/addlhelp/shim.py

```diff
@@ -219,6 +228,8 @@
 <B>BOTO CONFIGURATION</B>
   Configuration found in the boto file is mapped 1:1 to gcloud environment
   variables where appropriate.
+The ``gs_oauth2_refresh_token`` and ``gs_service_key_file`` fields are ignored
```

Comment:
Add a space to make this its own paragraph, or potentially add `Important: ` as a prefix so that it comes out looking like a call out (like this: https://screenshot.googleplex.com/B9yP3tcaAM8N2ag)

Reply:
Done